packages/button/theme/material/vaadin-button-styles.js

```diff
@@ -0,0 +1,183 @@
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import '@vaadin/vaadin-material-styles/color.js';
+import '@vaadin/vaadin-material-styles/shadow.js';
+import '@vaadin/vaadin-material-styles/typography.js';
+
+registerStyles(
+  'vaadin-button',
```

Comment:
Same as above (see comment for Lumo):

```suggestion
registerStyles(
  '',
```

Reply:
Done.